File: library/src/main/java/com/proxerme/library/enums/TagRateFilter.java
package com.proxerme.library.enums;

import com.squareup.moshi.Json;

/**
 * Enum for filtering rated tags.
 *
 * @author Ruben Gees
 */
public enum TagRateFilter {
    @Json(name = "rate_1")RATED_ONLY,
    @Json(name = "rate_10")ALL
}

File: library/src/main/kotlin/me/proxer/library/enums/TagRateFilter.kt
package me.proxer.library.enums

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

/**
 * Enum for filtering rated tags.
 *
 * @author Ruben Gees
 */
@JsonClass(generateAdapter = false)
enum class TagRateFilter {

    @Json(name = "rate_1")
    RATED_ONLY,

    @Json(name = "rate_10")
    ALL
}
